src/Confluent.Kafka/AdminClient.cs

```diff
@@ -657,6 +708,19 @@ public Task DeleteGroupsAsync(IList<string> groups, DeleteGroupsOptions options
             return completionSource.Task;
         }
 
+        /// <summary>
+        ///     Refer to <see cref="Confluent.Kafka.IAdminClient.DeleteConsumerGroupOffsetsAsync(String, IEnumerable{TopicPartition}, DeleteConsumerGroupOffsetsOptions)" />
+        /// </summary>
+        public Task<List<DeleteConsumerGroupOffsetsResult>> DeleteConsumerGroupOffsetsAsync(String group, IEnumerable<TopicPartition> partitions, DeleteConsumerGroupOffsetsOptions options = null)
```

Comment:
we don't need to return a list, as there is only one group.

Reply:
Updated